.github/workflows/gen_bindings.yml

```diff
@@ -66,6 +66,10 @@ jobs:
         with:
           repository: colinbellino/sokol-jai
           path: bindgen/sokol-jai
+      - uses: actions/checkout@main
+        with:
+          repository: radekm/sokol-c3
```

Comment:
This needs to be changed to floooh/sokol-c3.

And we also should have a build step `test-c3` further down which depends on the `gen-bindings` job. The test job should basically be a copy of the CI job in the sokol-c3 repo. It's just there to prevent pushing to sokol-c3 if test compilation fails.

(...I can take care of the final deploy-job, since this involves creating an SSH key and some Github shenanigans to allow pushing to the sokol-c3 repo)